crates/turbopack-cli-utils/src/issue.rs

```diff
@@ -570,3 +579,18 @@ fn show_all_message_with_shown_count(
         .bold()
     }
 }
+
+fn render_styled_string_to_ansi(styled_string: &StyledString) -> String {
+    match styled_string {
+        StyledString::Line(parts) => {
+            let mut string = String::new();
+            for part in parts {
+                string.push_str(&render_styled_string_to_ansi(part));
+            }
+            string
+        }
+        StyledString::String(string) => string.to_string(),
```

Comment:
What are the main differences between lines vs string?

Reply:
A line can contain many StyledStrings composed together into a line. It should also print a newline.